src/screens/TopicPageScreen/elements/TopicDomainHeader.js

```diff
@@ -88,13 +89,13 @@ const styles = StyleSheet.create({
     textAlign: 'left',
     color: isHeaderHide ? COLORS.white : COLORS.blackgrey
   }),
-  seeMemberText: {
+  seeMemberText: (followType) => ({
     fontSize: normalizeFontSize(12),
     fontFamily: fonts.inter[500],
     textAlign: 'left',
-    color: COLORS.blue,
+    color: followType === 'incognito' ? COLORS.anon_secondary : COLORS.blue,
     marginTop: normalize(1)
-  }
+  })
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The color of the text is being changed based on the `followType`. It's good that you're considering different styles for different modes. However, it would be better to define these colors in a constants file or theme file and then reference them here. This would make it easier to manage and change these colors in the future.
```diff
- color: followType === 'incognito' ? COLORS.anon_secondary : COLORS.blue,
+ color: followTypeColors[followType],
```
In the above suggestion, `followTypeColors` is an object defined in your constants or theme file like so:
```javascript
export const followTypeColors = {
  incognito: COLORS.anon_secondary,
  signed: COLORS.blue,
};
```
This way, if you ever need to add more follow types or change the colors, you can do it in one place.


<!-- This is an auto-generated comment by OSS CodeRabbit -->